OSX/Metabase/UI/MainViewController.m

```diff
@@ -133,8 +133,14 @@ - (void)saveCSV:(NSString *)apiURL {
 	if ([savePanel runModal] == NSFileHandlingPanelOKButton) {
 		NSLog(@"Will save CSV at: %@", savePanel.URL);
 		
-		NSURL *url = [NSURL URLWithString:apiURL relativeToURL:[NSURL URLWithString:BaseURL()]];
-		NSURLRequest *csvRequest = [NSURLRequest requestWithURL:url cachePolicy:NSURLRequestReloadIgnoringCacheData timeoutInterval:10.0f];
+		NSURL *url = [NSURL URLWithString:@"/api/dataset/csv" relativeToURL:[NSURL URLWithString:BaseURL()]];
+        NSString *postBody = [NSString stringWithFormat:@"query=%@",datasetQuery];
+        NSData *data = [postBody dataUsingEncoding:NSUTF8StringEncoding];
+		NSMutableURLRequest *csvRequest = [NSMutableURLRequest requestWithURL:url cachePolicy:NSURLRequestReloadIgnoringCacheData timeoutInterval:10.0f];
+        csvRequest.HTTPMethod = @"POST";
+        [csvRequest setValue:@"application/x-www-form-urlencoded" forHTTPHeaderField:@"Content-Type"];
```

Comment:
yeah, that's the way the endpoint is coded up because that's what we have to do in the web app.